R/paginate.R

```diff
@@ -135,7 +150,7 @@ paginate_req_perform <- function(req,
     resp <- req_perform(req)
     parsed <- parse_resp(resp)
 
-    out[[page]] <- parsed
+    out[[page]] <- parsed$data
```

Comment:
If we do this, I wonder if we should `vec_c()` the results together at the end. That will work for most "data" types in R, and if it doesn't work for you, you can always wrap in a `list()`.

Reply:
I like the idea and added this. It adds a dependency on vctrs but I think that vctrs is so commonly used anyways that it doesn't really matter much.